.gitignore

```diff
@@ -19,3 +19,8 @@ dist/
 
 # cypress
 cypress/
+
+# intellij
+.idea/
+*.iml
+
```

Comment:
could you please remove this unrelated change?

You can set global gitignore rules in `~/.gitignore` so you don’t need to set it per project :)

Reply:
yeah good point - want to leave it there for others, or remove?

Reply:
please remove, thank you!

Reply:
done